centaurCwlRunner/src/main/scala/centaur/cwl/OutputManipulator.scala

```diff
@@ -134,9 +127,8 @@ object OutputManipulator extends Poly1 {
       val defaultSize = sizeContent.orElse(sizeFile(path)).map(Json.fromLong).getOrElse(Json.Null)
       val size = valueOrNull("size", defaultSize)
 
-      val basename: Option[Json] = if (isInsideDirectory) {
+      val basename: Option[Json] =
```

Comment:
seems like this is no longer `Option`al?

Reply:
It is still optional because of `valueOrNull`